.gitpod.yml

```diff
@@ -0,0 +1,30 @@
+github:
```

Comment:
I don't like adding this new file to the root. In theory, gitpod should behave like bors and allow this file to live in the `.github` folder.

I would assume it's an advertising feature for gitpod to not allow it to live in a non-root folder, but that's a scummy practise. See also https://github.com/gitpod-io/gitpod/issues/729, which still doesn't suggest to use `.github`.

Reply:
Thanks @cart for putting a comment through.

I don't expect much movement though, unfortunately.

Reply:
Thanks @cart for putting a comment through.

I don't expect much movement though, unfortunately.